K8S/job_templates/CA-VICTORIA-K8S-TEST-T2_job.yaml

```diff
@@ -12,6 +12,18 @@ spec:
       containers:
         - name: job-container
           image: git.computecanada.ca:4567/rptaylor/misc/atlas-grid-almalinux9-localuser:20240124
+          securityContext:
+            privileged: false
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+              - ALL
+            runAsNonRoot: true
+            runAsGroup: 10000
+            runAsUser: 10000
+            readOnlyRootFilesystem: true
+            seccompProfile: 
+              type: Unconfined
```

Comment:
Yes, that is so that unprivileged users can launch Apptainer containers inside k8s pods.

An alternative might be if k8s supported user namespaces. Huh last time I checked this did not exist, but it looks like it's a new feature available in 1.30: https://kubernetes.io/blog/2024/04/22/userns-beta/